crates/dreammaker/src/constants.rs

```diff
@@ -444,8 +446,9 @@ impl Expression {
 }
 
 /// Evaluate an expression in the preprocessor, with `defined()` available.
-pub fn preprocessor_evaluate(location: Location, expr: Expression, defines: &DefineMap) -> Result<Constant, DMError> {
+pub fn preprocessor_evaluate(location: Location, expr: Expression, defines: &DefineMap, env_file: Option<&PathBuf>) -> Result<Constant, DMError> {
```

Comment:
This is technically an API break because it's a public function I think? I made it an Option but i think outside calls would still technically scream about the new argument.

Reply:
No problem